gnd/proguard-rules.pro

```diff
@@ -26,7 +26,7 @@
 
```

Comment:
Adding a placeholder comment to see if it will prevent PR from being inadvertently merged before #58.

Reply:
Didn't work :/